presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/QueryBuilder.java

```diff
@@ -211,11 +211,11 @@ private String toPredicate(String columnName, Domain domain, Type type, List<Typ
         checkArgument(domain.getType().isOrderable(), "Domain type must be orderable");
 
         if (domain.getValues().isNone()) {
-            return domain.isNullAllowed() ? quote(columnName) + " IS NULL" : "FALSE";
+            return domain.isNullAllowed() ? quote(columnName) + " IS NULL" : "1=0";
```

Comment:
maybe extract this as constant and some comment, to make it more readable and to prevent someone revert your change

Reply:
That would need to be someone in a hurry, i think.

Reply:
We can just add a comment above 
```java
// not all databases support booleans, so use 1=1 and 1=0 instead